openapi.yaml

```
@@ -0,0 +1,547 @@
+            schema:
+              type: object
+              properties:
+                question:
+                  type: string
+                  description: The message or question you want to pass to the model.
+                stream_data:
+                  type: string
+                  description: >
+                    If set to `true`, partial message deltas will be sent to reduce waiting time.
+                    Tokens will be streamed as data-only server-sent events, terminating with `[DONE]`.
+                  enum: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)